var/spack/repos/builtin/packages/vtk-m/package.py

```diff
@@ -185,7 +176,7 @@ def cmake_args(self):
                 options.append("-DVTKm_ENABLE_CUDA:BOOL=OFF")
 
             # hip support
-            if "+hip" in spec:
+            if "+rocm" in spec:
                 options.append("-DVTKm_NO_DEPRECATED_VIRTUAL:BOOL=ON")
                 options.append("-DVTKm_ENABLE_HIP:BOOL=ON")
```

Comment:
`VTKm_ENABLE_HIP` does not do anything. HIP is enabled when `VTKm_ENABLE_KOKKOS=ON` and Kokkos has HIP enabled.